modules/rhcos-enable-node-scan-scannerv4.adoc

```diff
@@ -0,0 +1,19 @@
+// Module included in the following assemblies:
+//
+// * operating/manage-vulnerabilities/scan-rhcos-node-host.adoc
+// * cloud_service/upgrading-cloud/upgrade-cloudsvc-roxctl.adoc
+:_mod-docs-content-type: PROCEDURE
+[id="rhcos-enable-node-scan-scannerv4_{context}"]
+= Enabling {op-system} node scanning with Scanner V4
+
+[role="_abstract"]
+If you use {ocp}, you can enable scanning of {op-system-first} nodes for vulnerabilities by using {rh-rhacs-first}.
+
+:FeatureName: RHCOS node scanning with Scanner V4
+include::snippets/technology-preview.adoc[]
+
+.Prerequisites
+* For scanning {op-system} node hosts of the secured cluster, you must have installed Secured Cluster services on {ocp} {ocp-supported-version} or later. For information about supported platforms and architecture, see the link:https://access.redhat.com/articles/7045053[{product-title} Support Matrix]. For life cycle support information for {product-title-short}, see the link:https://access.redhat.com/support/policy/updates/rhacs[{product-title} Support Policy].
+//what action does user have to take to enable Scanner V4 for node scanning?
```

Comment:
In addition to having Secured Cluster Services deployed, it is also a hard requirement that Scanner V4 is installed and functional.
To enable Node Indexing (Node Scanning on Scanner V4), an environment variable needs to be set on Central and on Compliance:
- `kubectl -n stackrox set env deployment/central ROX_NODE_INDEX_ENABLED=true`
- `kubectl -n stackrox set env daemonset/collector ROX_NODE_INDEX_ENABLED=true`

We're currently not exposing the status of enabled features in the UI.
To verify that the feature is working as expected, Central logs can be checked for the message
`Scanned index report and found XXX components for node ABC` (listing the number of discovered components for a given node).

Reply:
Thinking about it, we should also state that ACS 4.6 is the minimum required version, right?